install/kubernetes/templates/istio-ingress.yaml.tmpl

```diff
@@ -70,6 +70,26 @@ spec:
         ports:
         - containerPort: 80
         - containerPort: 443
+        readinessProbe:
+          exec:
+            command:
+            - /bin/sh
+            - -c
+            - curl http://localhost:15000/clusters && curl http://localhost:15000/listeners && curl http://localhost:15000/routes
```

Comment:
why not use the standard http probe?

Reply:
Right, this is another option, but we are running out of time for Monday morning.

Reply:
Actually, this requires exposing port 15000. It is not a better option.

Reply:
I think performing checks without opening up new ports is preferred.

Reply:
the port is a good argument, I don't know how to configure it so it's only reachable from the probe so for readiness maybe this is a good approach, *if* it was actually checking something (which it isn't except for something listening on 15000)